plugins/inputs/snmp/README.md

```diff
@@ -100,8 +94,7 @@ To collect SNMP tables, use the `table` option.
 
 #### Field
 
-Use a `field` to collect a variable by OID.  Requests specified with this
-option operate similar to the `snmpget` utility.
+Use a `field` to collect a variable by OID.  
```

Comment:
Just to be clear, they are described that way because (even with the descriptions) people are sometimes having hard time to figure out when to use `Field` or `Table`.

Reply:
I removed it as we are no longer calling `snmpget` but I put it back for the sake of clarity.